src/app/query-panel/query-panel.component.html

```diff
@@ -18,15 +18,32 @@
     <!--    OSM STATS-->
     <div class="ui bottom attached tab secondary segment" data-tab="tab-ohsome-api"
          [ngClass]="activeBackend === 'ohsomeApi' ? ' active' : ''">
-      <app-ohsome-api-query-form *ngIf="activeBackend==='ohsomeApi'" [hashParams]=hashParams></app-ohsome-api-query-form>
+      @if (activeBackend === 'ohsomeApi' && ohsomeApiMetadataProviderService.ohsomeApiAvailable) {
+        <app-ohsome-api-query-form [hashParams]=hashParams></app-ohsome-api-query-form>
+      }
+      @if (activeBackend === 'ohsomeApi' && !ohsomeApiMetadataProviderService.ohsomeApiAvailable) {
+        <div class="ui one column grid">
+          <div class="row">
+            <div class="column">
+              <h4 class="ui header"><i class="exclamation triangle icon"></i>OSM History Stats based on ohsome API</h4>
+              <p>Sorry.</p>
+              <p>This service is temporarily unavailable.</p>
+              <p>Please try again later.</p>
+              <p>Thanks for your patience.</p>
+            </div>
+          </div>
+        </div>
+      }
     </div>
     <!--    OSM ANALYSIS-->
     <div class="ui bottom attached tab secondary segment" data-tab="tab-oqt-api"
          [ngClass]="activeBackend === 'oqtApi' ? ' active' : ''">
-      <app-oqt-api-query-form *ngIf="activeBackend==='oqtApi' && oqtApiMetadataProviderService.oqtApiAvailable" [hashParams]=hashParams (changeIndicatorCoverages)="onChangeIndicatorCoverages($event)"></app-oqt-api-query-form>
+      <app-oqt-api-query-form *ngIf="activeBackend==='oqtApi' && oqtApiMetadataProviderService.oqtApiAvailable && ohsomeApiMetadataProviderService.ohsomeApiAvailable"
```

Comment:
Is it on purpose that you also check the ohsome API metadata for the ohsome quality API (OQAPI) form? Theoretically the ohsome API can be (temporarily) restricted to the outside but not to OQAPI. Some OQAPI indicators do not depend on the ohsome API.

Reply:
yes it was on purpose. Currently it's only one (soon two) topics with one indicator that would work. All other topics with the default indicator set to mapping saturation would just fail. I think it currently better to tell the users that maintenance is going on an not letting them search the needle in the haystack. Not sure what error message oqapi sends but probably some backend failure. I thought it would be better to inform the user directly that the service is not working properly for a while.